test/unittest/box-cc0544b6afd1.lua

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env tarantool
```

Comment:
Commit hash used in a filename looks non-existent.

Reply:
Non-existent?

I guess the hash is needed to find a process using `ps`. However, yep, it is not very obvious and it would be good to comment it here.

Reply:
I added a comment about such names inside the test.

Reply:
> Non-existent?

Yep. I thought the idea was to add a hash with commit to filename, like we do with tests for tarantool (like `gh-6028-replica.lua`).